core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceMediaPlayer.java

```diff
@@ -255,13 +272,33 @@ private void playMediaObject(final Playable playable, final boolean forceReset,
         }
     }
 
-    private MediaMetadataCompat getMediaSessionMetadata(Playable p) {
-        MediaMetadataCompat.Builder builder = new MediaMetadataCompat.Builder();
-        builder.putString(MediaMetadataCompat.METADATA_KEY_ARTIST, p.getFeedTitle());
-        builder.putString(MediaMetadataCompat.METADATA_KEY_TITLE, p.getEpisodeTitle());
-        builder.putString(MediaMetadataCompat.METADATA_KEY_DISPLAY_TITLE, p.getEpisodeTitle());
-        builder.putString(MediaMetadataCompat.METADATA_KEY_ALBUM, p.getFeedTitle());
-        return builder.build();
+    private void updateMediaSessionMetadata() {
+        executor.execute(() -> {
+            Playable p = this.media;
+            MediaMetadataCompat.Builder builder = new MediaMetadataCompat.Builder();
+            builder.putString(MediaMetadataCompat.METADATA_KEY_ARTIST, p.getFeedTitle());
```

Comment:
I'm sometimes getting null pointer exceptions when this line is executed in tests (after rebasing on develop).